src/lib/ui/transcription-view.ts

```diff
@@ -0,0 +1,48 @@
+import prettyMilliseconds from 'pretty-ms'
```

Comment:
I prefer to write the helper method if it is easy and straightforward instead of adding a dependency.

Reply:
Even if we want to recreate that package, we'll have to use one of it's dependencies called `parse-ms` to make parsing the milliseconds easy.